docs/source/tutorials/base/how_to_train/detection.rst

```diff
@@ -0,0 +1,385 @@
+                            --test-data-roots data/wgisd/data 
+                            --load-weights outputs/weights.pth
+                            --save-performance outputs/performance.json
+  
+
+If you created ``data.yaml`` file in previous step, you can simplify the training by passing it in ``--data`` parameter. 
+Note,  with ``data.yaml``, it runs evaluation on test JSON annotation file (not on validation JSON annotation file). 
+
+.. code-block::
+
+  (detection) ...$ otx eval otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml
+                            --data data.yaml 
+                            --load-weights outputs/weights.pth
+                            --save-performance outputs/performance.json
+
+We will get this validation output:
+
+.. code-block::
+
+  2023-01-10 06:17:32,182 | INFO : run task done.
+  2023-01-10 06:17:33,884 | INFO : Inference completed
+  2023-01-10 06:17:33,885 | INFO : called evaluate()
+  2023-01-10 06:17:34,142 | INFO : F-measure after evaluation: 0.5487693710118504
+  2023-01-10 06:17:34,142 | INFO : Evaluation completed
+  Performance(score: 0.5487693710118504, dashboard: (1 metric groups))
+
+3. The output of ``./outputs/performance.json`` consists of dict with target metric name and its value.
+
+.. code-block::
+
+  {"f-measure": 0.5487693710118504}
+
+4. ``Optional`` Additionally, we can tune evaluation parameters such as confidence threshold via the command line. Read more about template-specific parameters for validation in quick start [#TODO link].
+
+For example, if there are too many False-Positive predictions (there we have a prediction, but don't have annotated object for it) can suppress its number by increasing the confidence threshold as it is shown below.
+
+Please note, by default, the optimal confidence threshold is detected based on validation results to maximize the final F1 metric. To set a custom confidence threshold, please disable ``result_based_confidence_threshold`` option.
+
+.. code-block::
+
+  (detection) ...$ otx eval otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml
+                            --data data.yaml 
+                            --load-weights outputs/weights.pth
+                            params 
+                            --postprocessing.confidence_threshold 0.5
+                            --postprocessing.result_based_confidence_threshold false 
+
+  ...
+
+  2023-01-10 06:21:04,254 | INFO : F-measure after evaluation: 0.514346439957492
+
+*********
+Export
+*********
+
+1. ``otx export`` exports a trained Pytorch `.pth` model to the OpenVINO™ Intermediate Representation (IR) format. 
+It allows to run the model on the Intel hardware much more efficient, especially on the CPU. Also, the resulting IR model is required to run POT optimization in the section below. IR model consists of 2 files: ``openvino.xml`` for weights and ``openvino.bin`` for architecture.
```

Comment:
Much more efficient then what?

Reply:
than Pytorch `.pth` model. It follows from the previous sentence.

Reply:
The previous sentence just says that there are 2 model formats (although .pth is not really a format). Then, actually it's openvino runtime allows efficient inference of IRs, not the IR format itself. And I think we don't need an article before CPU, since we are not speaking about a particular one, but rather about Intel CPUs as a class of devices in general.

Reply:
I'll specify that OpenVINO runtime helps run the model faster, not the format itself,  but in my next documentation PR, if you don't mind. Thank you, this specification is really valuable